data/README.md

```diff
@@ -96,3 +96,78 @@ AWS_CDN=$(aws --profile=iow-boundary-tool cloudfront list-distributions --query
 
 aws --profile=iow-boundary-tool cloudfront create-invalidation --distribution-id $AWS_CDN --paths "/example/*"
 ```
+
+## Importing Contributors
+
+While individual Contributors can be created via the Django Admin, that can be tedious when deealing with a large set of new users.
```

Comment:
```suggestion
While individual Contributors can be created via the Django Admin, that can be tedious when dealing with a large set of new users.
```

Reply:
Fixed in [dcc66e3](https://github.com/azavea/iow-boundary-tool/pull/238/commits/dcc66e33740c07c7ffe88fcdd0aa64dde60053a6)